.github/workflows/release.yml

```diff
@@ -36,7 +38,6 @@ jobs:
       uses: pypa/gh-action-pypi-publish@release/v1
       with:
         user: __token__
-        password: ${{ secrets.PYPI_TOKEN }}
```

Comment:
We can remove all of these lines, right?

Reply:
Yep! I missed that last one; just removed.